src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/NotificationService.java

```diff
@@ -94,11 +94,16 @@ private void sendContestedApplicationIssuedEmail(NotificationRequest notificatio
         sendNotificationEmail(notificationRequest, uri);
     }
 
-    public void sendContestOrderApprovedEmail(CallbackRequest callbackRequest) {
+    public void sendContestOrderApprovedEmailApplicant(CallbackRequest callbackRequest) {
         URI uri = buildUri(notificationServiceConfiguration.getContestOrderApproved());
         sendNotificationEmail(notificationRequestMapper.createNotificationRequestForAppSolicitor(callbackRequest), uri);
     }
 
+    public void sendContestOrderApprovedEmailRespondent(CallbackRequest callbackRequest) {
+        URI uri = buildUri(notificationServiceConfiguration.getContestOrderApproved());
+        sendNotificationEmail(notificationRequestMapper.createNotificationRequestForRespSolicitor(callbackRequest), uri);
```

Comment:
have a look at other PRs; we have a base sendContestOrderApprovedEmail which will take the NotificationRequest object and generate the URI and call send notification (means we're only doing this in one place)